app/controllers/issues_controller.rb

```diff
@@ -32,6 +39,12 @@ def create
     end
   end
 
+  def map_issue
+    @issue = Issue.find(params[:id])
+    @response = { issue: @issue, img: @issue.issue_attachments.first }
```

Comment:
what if no images? try and fix

ALWAYS try to handle case when something could be nil